pkgs/development/libraries/wayland/default.nix

```diff
@@ -10,7 +10,8 @@
 , libxml2
 , wayland
 , expat ? null # Build wayland-scanner (currently cannot be disabled as of 1.7.0)
```

Comment:
Is this still right? Can we remove the null or replace it with a real boolean flag if it is useful?

Reply:
I'll try to create a follow-up PR with some other changes as this seems out of scope. I don't think the comment makes sense anymore (we need wayland-scanner and could only consider splitting wayland-scanner into a dedicated package but this would be difficult to do without breaking packages or running a mass-rebuild for testing).

Reply:
I've removed the `null` part in ea099211e0c4dc8163ad6cbda3abddd3e75bf5b4 and submitted https://gitlab.freedesktop.org/wayland/wayland/-/merge_requests/128 to make it "optional" (but this currently doesn't help us as we'd need a separate `wayland-scanner` package).